cli/src/common_utils/utility_scripts.sh

```diff
@@ -44,10 +46,39 @@ function get_pillar_data {
         exit 1
     fi
 
-    local l_val=$(salt-call pillar.get ${l_key} --output=newline_values_only)
+    local l_val=$(salt-call --local pillar.get "${l_key}" --output=newline_values_only '${SALT_OPTS}')
```

Comment:
I believe you understand the limitations of that approach and how it differs from `salt <target> pillat.get`

Reply:
Yes, there would be limitations.
This is targeted for use only during boxing and esp. unboxing.
The challenge is in environments where gluster is offline, the data fails back to not brick directory but the /opt/seagate directory for pillar, which is my concern.